doc/source/actions.rst

```diff
@@ -201,3 +201,17 @@ The ``merge`` action merges the pull request into its base branch. The
        again. If you prefer to update one pull request at a time (for example,
        to save CI runtime), set ``strict`` to ``smart`` instead: Mergify will
        queue the mergeable pull requests and update them one at a time serially.
+   * - ``strict_method``
+     - string
+     - ``merge``
+     - Base branch update method when strict mode is enabled.
+       Possible values are ``merge`` or ``rebase``.
+       Note that ``rebase`` have many drawback due to the change of all commits
+       sha of the pull request, some example:
+
+       * Your contributor will need to "force push" it's own branch if it adds new commits.
+       * GitHub branch protection of your project may dismiss approved reviews.
+       * GitHub branch protection of the contributor may refuse Mergify to
```

Comment:
Branch protection of the contributor?

Reply:
I don't think people use this, but you can add branch protection on your fork and mess up upstream bot.